packages/coordinator/src/screens/AuthScreens/ResetPasswordScreen.tsx

```diff
@@ -1,28 +1,36 @@
 import { useState } from "react";
 import styles from "./AuthScreens.module.scss";
 import Input from "../../components/Input";
-import Button from "../../components/Button";
+import Button, { ButtonVariant } from "../../components/Button";
+import emailNotFound from "../../assets/email_not_found.svg";
+import emailOnTheWay from "../../assets/email_on_the_way.svg";
 import logoImage from "./../../assets/logo_drop_ZM.svg";
 import chevronSvg from "./../../assets/icons/chevron-right-small.svg";
 import { useNavigate } from "react-router-dom";
+import { Popup } from "../../components/Popup/Popup";
 
 export interface ResetPasswordScreenProps {
-  onResetPassword: (
-    email: string,
-    emailError: (error: string) => void
-  ) => Promise<void>;
+  onResetPassword: (email: string, emailError: (error: string) => void) => void;
 }
 
 export default function ResetPasswordScreen(props: ResetPasswordScreenProps) {
   const [email, setEmail] = useState("");
   const [emailError, setEmailError] = useState("");
   const [isLoading, setIsLoading] = useState(false);
 
+  const [successPopupOpen, setSuccessPopupOpen] = useState<boolean>(false);
+  const [errorPopupOpen, setErrorPopupOpen] = useState<boolean>(false);
+
   const navigate = useNavigate();
 
-  const resetPassword = async () => {
+  const handleResetPass = async () => {
     setIsLoading(true);
-    await props.onResetPassword(email, setEmailError);
+    try {
+      await props.onResetPassword(email, setEmailError);
```

Comment:
Please verify this works locally  - try sending a reset email to a broken email and see the popup works

Reply:
fixed and checked